### PR TITLE
Bump a bunch of dependencies to more recent versions

### PR DESCRIPTION
**Pull request by vdemeester**
A bit related to https://github.com/docker/cli/pull/1230 (i.e. make more of the common deps referering to the same version)

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Review by boaz0 — APPROVED** [review]
LGTM :rocket: if all :green_apple:

**Comment by cpuguy83** [comment]
Looks legit: 

```
13:45:37 ----------------------------------------------------------------------
13:45:37 FAIL: docker_api_test.go:91: DockerSuite.TestAPIErrorNotFoundJSON
13:45:37 
13:45:37 docker_api_test.go:95:
13:45:37     c.Assert(httpResp.StatusCode, checker.Equals, http.StatusNotFound)
13:45:37 ... obtained int = 405
13:45:37 ... expected int = 404
```

**Comment by vdemeester** [comment]
@cpuguy83 yep :sweat: gonna look at it :sweat_smile:

**Comment by ripcurld** [comment]
good job! :crossed_fingers:

**Review by thaJeztah — APPROVED** [review]
LGTM if green

**Comment by ripcurld** [comment]
@thaJeztah @vdemeester it seems like the failure isn't related:
```shell
13:42:24 FAIL: docker_api_swarm_service_test.go:311: DockerSwarmSuite.TestAPISwarmServicesFailedUpdate
13:42:24 
13:42:24 [d238601374b28] waiting for daemon to start
13:42:24 [d238601374b28] daemon started
13:42:24 
13:42:24 [d7b665da69c27] waiting for daemon to start
13:42:24 [d7b665da69c27] daemon started
13:42:24 
13:42:24 [d768e59ef6e25] waiting for daemon to start
13:42:24 [d768e59ef6e25] daemon started
13:42:24 
13:42:24 docker_api_swarm_service_test.go:347:
13:42:24     waitAndAssert(c, defaultReconciliationTimeout, daemons[0].CheckRunningTaskImages, checker.DeepEquals,
13:42:24         map[string]int{image1: instances})
13:42:24 docker_utils_test.go:435:
13:42:24     c.Assert(v, checker, args...)
13:42:24 ... obtained map[string]int = map[string]int{"busybox:latest":4}
13:42:24 ... expected map[string]int = map[string]int{"busybox:latest":5}
13:42:24 
13:42:24 [d238601374b28] exiting daemon
13:42:24 [d7b665da69c27] exiting daemon
13:42:24 [d768e59ef6e25] exiting daemon
13:42:49 
```

File: vendor.conf [hunk 1]
```diff
@@ -55,9 +55,9 @@ github.com/vishvananda/netlink b2de5d10e38ecce8607e6b438b6d174f389a004e
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc374
 github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
-github.com/coreos/etcd v3.2.1
+github.com/coreos/etcd v3.3.9
```
Comment (cpuguy83) [inline vendor.conf]: Is it wise to update to 3.3 here?
Swarmkit still only has 3.2.1.

Comment (vdemeester) [reply vendor.conf]: Yeah, we discussed this with @thaJeztah not sure either :sweat: 
I can remove the bump from there and try to update to latest `etcd` in `swarmkit` and `moby` :angel:

**Comment by thaJeztah** [comment]
@vdemeester needs a rebase

**Pull request merged**